src/Components/ToDo.tsx

```diff
@@ -0,0 +1,21 @@
+import { ToDoProps } from "../Shared/interface";
```

Comment:
instead of interface use types name for the file
"../Shared/interface"

Reply:
Done